NEWS.md

```diff
@@ -13,6 +13,7 @@
 * Nested calls to `across.()` are handled properly (#505)
 * `across.()`: Can namespace functions in `.fns` arg (#511)
 * `as_tidytable()`: Can keep row names when converting a matrix (#527)
+* `unnest.()`: Handles empty tibbles/tidytables (#530)
```

Comment:
Can we drop the `tibbles/tidytable` and just say `data frames` instead? That shows it generalizes to all data frame objects (tibble/tidytable/data.table).

And can you put `(@roboton, #530)` - that way people will know it was your update that fixed the issue. Sorry I forgot to mention this part!

Reply:
Done, just FYI unnest.() will still throw an error for data.table and data.frame since it can't reference an empty column:

```
Error (test-unnest.R:156:3): handles empty data frames
<vctrs_error_subscript_oob/vctrs_error_subscript/rlang_error/error/condition>
Error in `chr_as_locations(x, vars, call = call)`: Can't subset columns that don't exist.
x Column `x` doesn't exist.
```

```
> data.frame(x = list())$x
NULL
> data.table::data.table(x = list())$x
NULL
> tidytable::tidytable(x = list())$x
list()
> tibble::tibble(x = list())$x
list()
```

Reply:
It should still work - the first step of a tidytable function is to convert a data frame input to a `tidytable`.

Using this PR:

``` r
devtools::load_all(".")
#> ℹ Loading tidytable

df <- data.table(id = integer(), foo = list())

df %>%
  unnest.(foo)
#> # A tidytable: 0 × 2
#> # … with 2 variables: id <int>, foo <lgl>
```

Reply:
Interesting... actually this fails when `foo` is the only column.

Reply:
Good catch - can you change this back to ``* `unnest.()`: Handles empty tibbles/tidytables``?